.github/workflows/pr-checks.yaml

```diff
@@ -90,44 +85,53 @@ jobs:
-          chmod +x operator-sdk
-          mv -v operator-sdk $GITHUB_WORKSPACE/bin/
+          eval $(minikube docker-env)
+          make OPERATOR_REPO_REF=$(minikube ip):5000/sbo SKIP_REGISTRY_LOGIN=true release-operator -o registry-login test-acceptance-with-bundle
 
-          curl -Lo opm https://github.com/operator-framework/operator-registry/releases/download/v${OPM_VERSION}/linux-amd64-opm
-          chmod +x opm
-          mv -v opm $GITHUB_WORKSPACE/bin/
 
-          curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v${K8S_VERSION}/bin/linux/amd64/kubectl
-          chmod +x kubectl
-          mv -v kubectl $GITHUB_WORKSPACE/bin/
+  acceptance_without_olm:
+    name: Acceptance tests running on Kubernetes without using OLM
+    runs-on: ubuntu-20.04
 
-          curl -Lo minikube https://storage.googleapis.com/minikube/releases/v${MINIKUBE_VERSION}/minikube-linux-amd64
-          chmod +x minikube
-          mv -v minikube $GITHUB_WORKSPACE/bin/
+    env:
+      EXTRA_BEHAVE_ARGS: "--tags=~@knative --tags=~@openshift --tags=~@olm"
+      UMOCI_VERSION: "0.4.5"
+      OPERATOR_VERSION: "0.4.0"
 
-      - name: Set KUBECONFIG
-        run: |
-          KUBECONFIG=$GITHUB_WORKSPACE/miniconfig
-          echo "KUBECONFIG=$KUBECONFIG" >> $GITHUB_ENV
-          touch $KUBECONFIG
+    steps:
+      - name: Checkout Git Repository
+        uses: actions/checkout@v2
 
-      - name: Set up minikube
-        run: |
-          ./hack/start-minikube.sh start --kubernetes-version=v${K8S_VERSION} --driver=$CONTAINER_RUNTIME --cpus $(nproc) --memory 5g
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: "^1.15.6"
 
-      - name: Docker Info
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.7"
+          architecture: "x64"
+
+      - name: Set up CLI
+        uses: ./.github/actions/setup-cli
+
+      - name: Setup umoci cli
         run: |
-          eval $(minikube docker-env)
-          docker ps
-          kubectl get nodes -o yaml
-          kubectl cluster-info
-          docker info
+          curl -Lo umoci https://github.com/opencontainers/umoci/releases/download/v${UMOCI_VERSION}/umoci.amd64
+          chmod +x umoci
+          mv -v umoci $GITHUB_WORKSPACE/bin/
 
-      - name: Acceptance tests
+      - name: Acceptance tests against vanilla k8s without OLM
         timeout-minutes: 60
         run: |
           eval $(minikube docker-env)
-          make OPERATOR_REPO_REF=$(minikube ip):5000/sbo SKIP_REGISTRY_LOGIN=true release-operator -o registry-login test-acceptance-with-bundle
\ No newline at end of file
+          make OPERATOR_REPO_REF=$(minikube ip):5000/sbo SKIP_REGISTRY_LOGIN=true release-operator -o registry-login release-manifests
+          kubectl apply -f out/release.yaml
```

Comment:
all resources that are not cluster-scoped should be deployed within `service-binding-operator` namespace. Hence, `release.yaml` needs to contain `Namespace` object and all other namespace-scoped objectes should have `namespace` seto `service-binding-operator`. By inspecting the python script, I can see this is not the case.